pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -0,0 +1,264 @@
+import logging
```

Comment:
Please search in the entire codebase for `run`. Most likely all of them have to be replaced with `task` instead.

Didn't review this very well. Please provide integration tests first (as in proWES).